src/client/components/vision/camera/shaders/world_line.frag

```diff
@@ -121,36 +193,42 @@ float angleAround(vec3 axis, vec3 start, vec3 vec) {
 
 void main() {
 
-  // Get our position on the screen in pixel coordinates
-  vec2 screenPoint = vec2(0.5 - vUv.x, vUv.y - 0.5) * viewSize;
+  // Get our position on the screen
+  vec2 point = vec2(0.5 - vUv.x, vUv.y - 0.5);
```

Comment:
This is not equivalent, you need to fix up the aspect ratio by multiplying by viewSize and then dividing by viewSize.x